melosys-eessi-test/src/test/java/no/nav/melosys/eessi/SedMottakTestIT.java

```diff
@@ -169,25 +169,98 @@ void sedMottattIkkeIdentifisert_oppgaveBlirIdentifisertOgMarkertSomFeilIdentifis
 
         //Forventer kun én melding, som er oppgave-endret record
         kafkaTestConsumer.reset(1);
-        kafkaTemplate.send(lagOppgaveIdentifisertRecord(oppgaveID)).get();
+        kafkaTemplate.send(lagOppgaveIdentifisertRecord(oppgaveID, FNR ,"1")).get();
         kafkaTestConsumer.doWait(5_000L);
 
         verify(oppgaveConsumer, timeout(4000)).oppdaterOppgave(eq(oppgaveID), any());
 
         assertThat(hentMelosysEessiRecords()).isEmpty();
     }
 
-    private ProducerRecord<String, Object> lagOppgaveIdentifisertRecord(String oppgaveID) {
-        return new ProducerRecord<>("oppgave-endret", "key", oppgaveEksempel(oppgaveID, FNR, AKTOER_ID));
+    @Test
```

Comment:
Testen `SedMottakIT` her tester mottak av SED-er. Jeg synes det ville være bedre å opprette en egen testklasse for testing av mottak av oppgave-hendelser.

Reply:
👍